work/service/work_handlers.go

```diff
@@ -0,0 +1,85 @@
+package service
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	dataService "github.com/tidepool-org/platform/data/service"
+	request "github.com/tidepool-org/platform/request"
+	workLoad "github.com/tidepool-org/platform/work/load"
+)
+
+func LoadTestRoutes() []dataService.Route {
+	return []dataService.Route{
+		dataService.Get("/v1/work/load/ok", okHandler),
```

Comment:
I would use `/v1/work/load/status` as it is more intuitive since you are returning the status.